hack/release/common.sh

```diff
@@ -109,7 +126,7 @@ cosignImages() {
 pullPrivateReplica(){
```

Comment:
I'm assuming that this is happening in a follow-on, but we can drop this function later.